### ISSUE TITLE
Module 'vue' has no exported Member 'Ref'

### ISSUE THREAD

**Issue by philipp-kremer** [prazdevs/pinia-plugin-persistedstate#90]
Hey Guys,

i ran into following error after using `pinia-plugin-persistedstate`

``` bash
ERROR in ../node_modules/pinia-plugin-persistedstate/dist/index.d.ts(2,10):
2:10 Module '"vue"' has no exported member 'Ref'. Did you mean to use 'import Ref from "vue"' instead?
    1 | import { StateTree as StateTree$1, PiniaPluginContext } from 'pinia';
  > 2 | import { Ref } from 'vue';
      |          ^
    3 | 
    4 | interface CookieOptions<T> {
    5 |     decode?(value: string): T;
Version: typescript 4.4.4
Time: 4223ms
```

I'm currently using Vue 2 with the `@vue/composition-api` plugin. And installed your plugin like this:

`pinia.ts`
``` typescript
import Vue from 'vue';
import { createPinia, PiniaVuePlugin } from 'pinia';
import { createPersistedState } from 'pinia-plugin-persistedstate';
import { PINIA_STORAGE } from '@/utils/const';

Vue.use(PiniaVuePlugin);

const pinia = createPinia();

pinia.use(
  createPersistedState({
    storage: PINIA_STORAGE,
  })
);

export default pinia;
```

`main.ts`
``` typescript
...
// Make sure that composition api and pinia are imported first
import '@/plugins/composition-api';
import pinia from '@/plugins/pinia';
...

new Vue({
  ...
  pinia,
  ...
  render: (h) => h(App),
}).$mount('#app');

```

implemented in my stores i did it like this:
`auth.ts`
``` typescript
persist: {
    key: PINIA_PERSIST_KEY,
  },
```

My Project is working fine, the store is persited and everything seems fine, but i want to remove the error message from my command line.
Is this an error created by me or is it related to something else?

**Comment by prazdevs** [comment]
Ah i see what it is, I havent played at all with a Vue2 project with TS. And it comes down to all the types being in one place.
V2 should fix this as the core plugin will not have any dependency at all and not bother you with this.

the Ref import is used to define the Nuxt helper typings. In V2, nuxt specific imports will be in a separate file.

tl;dr: dont bother, problem comes from the plugin, and shouldnt cause you any issue besides the warning. it'll also be fixed in V2

**Comment by philipp-kremer** [comment]
Thanks for the quick answer!

Is there already some timeline when to release v2 or is it possible to push a hotfix for that?

I would really appreciate that so i can integrate it without failing in jenkins

**Comment by prazdevs** [comment]
well since v2 API is hopefully 100% iso with current v1 API, I _could_ release it. just needs the more complete docs. 
ETA would be smth like this weekend maybe ?

**Comment by philipp-kremer** [comment]
Would be awesome. Thank You!
